8Knot/pages/affiliation/visualizations/org_associated_activity.py

```diff
@@ -221,8 +221,8 @@ def process_data(df: pd.DataFrame, num, start_date, end_date, email_filter):
     # creates list of emails for each contribution and flattens list result
     emails = df.email_list.str.split(" , ").explode("email_list").tolist()
 
-    # remove any entries not in email format
-    emails = [x for x in emails if "@" in x]
+    # remove any entries not in email format and flattens list result
+    emails = [x.lower() for x in emails if "@" in x]
```

Comment:
noticing this for the first time- might prefer to use a regular expression for email correctness, rather than just checking for the `@` sign.

Reply:
What do you see the regex providing that this solution does not?

Reply:
if we're seeing "emails" that aren't actually email format, we can use the canonical "is this an email" regex to check for the real email rules. If we don't really care about that level of formalism, no need to pursue it.

Reply:
Yeah I think at the current state that level of formalism is above what we are looking for